src/main/resources/exclusions.properties

```diff
@@ -0,0 +1,3 @@
+# Multiple mechanisms present that use the same credential transport HttpCredentialTransport. 
+# Mechanisms are io.quarkus.smallrye.jwt.runtime.auth.JWTAuthMechanism and io.quarkus.oidc.runtime.OidcAuthenticationMechanism
+smallrye-jwt,oidc=true
```

Comment:
Each exclusion should have information about why are incompatible with each other.